doc/src/features/Praktikumsplaetze.md

```diff
@@ -133,6 +133,17 @@ A green notification appears if it was successful, if not a red error will appea
 ## Deletion
 It is possible to delete [Praktikumsstellen](../glossary.md#praktikumsstelle) in the overview of [Praktikumsstellen](../glossary.md#praktikumsstelle) by clicking the delete button, marked with a trash can.
 
+## Modify
+
+Already created [Praktikumsstellen](../glossary.md#praktikumsstelle) can be modified. By clicking the "Bearbeiten" button with the pencil icon on a [Praktikumsstellen](../glossary.md#praktikumsstelle) card in the [Praktikumsstellen](../glossary.md#praktikumsstelle) overview you can edit a [Praktikumsstelle](../glossary.md#praktikumsstelle) in the current [Meldezeitraum](../glossary.md#meldezeitraum).
```

Comment:
With that doc i expected the edit button within the view i reach after clicking on `Praktikumsplätze`. But the edit button ist located in the assign view after expanding a Praktikumsstelle

Reply:
See lines 140 to 145 for and additional description to edit Praktikumsstellen in the assign view. Although I'll add the information with the expand button.